test/Microsoft.AspNetCore.SignalR.Client.Tests/ConnectionTests.cs

```diff
@@ -479,7 +480,7 @@ public void ConnectionReturnsUrlUsedToStartTheConnection()
                 .Returns<HttpRequestMessage, CancellationToken>(async (request, cancellationToken) =>
                 {
                     await Task.Yield();
-                    if (request.RequestUri.AbsolutePath.EndsWith("/send"))
+                    if (request.Method == HttpMethod.Post)
```

Comment:
That's not necessary, the client has an HttpClient enum and that's what's being used here.